CRUD-APP/App.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './App.css';
+import '../node_modules/bootstrap/dist/css/bootstrap.css';
+import Home from './components/Home';
+import About from './components/About';
+import Contact from './components/Contact';
+import Navbar from './layout/Navbar';
+import Error from './components/Error';
+import {BrowserRouter, Route, Switch} from 'react-router-dom';
+import AddUser from './user/AddUser';
+import EditUser from './user/EditUser';
+import ViewUser from './user/ViewUser';
```

Comment:
try to implement React lazy loading
```suggestion
import AddUser from './user/AddUser';
import EditUser from './user/EditUser';
import ViewUser from './user/ViewUser';
```